src/frontends/tensorflow/src/op/unpack.cpp

```diff
@@ -19,9 +19,17 @@ OutputVector translate_unpack_op(const NodeContext& node) {
     auto num = node.get_attribute<int64_t>("num");
```

Comment:
What if `axis` is not specified shouldn't we use zero value by the specification?